saleor/graphql/warehouse/mutations.py

```diff
@@ -0,0 +1,84 @@
+import graphene
+
+from saleor.account.models import Address
+from saleor.graphql.core.mutations import ModelDeleteMutation, ModelMutation
+from saleor.graphql.core.types.common import WarehouseError
+from saleor.graphql.warehouse.types import WarehouseCreateInput, WarehouseUpdateInput
+from saleor.warehouse import models
+
+ADDRESS_FIELDS = [
+    "street_address_1",
+    "street_address_2",
+    "city",
+    "city_area",
+    "postal_code",
+    "country",
+    "country_area",
+    "phone",
+]
+
+
+class WarehouseCreate(ModelMutation):
+    class Arguments:
+        input = WarehouseCreateInput(
+            required=True, description="Fields required to create warehouse."
+        )
+
+    class Meta:
+        description = "Creates new warehouse."
+        model = models.Warehouse
+        permissions = ("warehouse.manage_warehouses",)
+
+    @classmethod
+    def create_address(cls, clean_data):
+        address_data = clean_data["address"]
```

Comment:
Do we validate `address` somehow? 
I see that some address mutations use address validation from `I18nMixin`.